kubernetes/internal/generate-weblogic-operator-cert.sh

```diff
@@ -39,6 +40,9 @@ OP_KEY_PEM="${CERT_DIR}/${OP_PREFIX}.key.pem"
 rm -rf ${CERT_DIR}
 mkdir ${CERT_DIR}
 
+# TBD - validate that keytool and openssl are available
```

Comment:
What's the plan here?

Reply:
How about file a jira to add these checks and remove this comment?

Reply:
Ok